server/mailet/mock-smtp-server/src/test/java/org/apache/james/mock/smtp/server/OperatorTest.java

```diff
@@ -0,0 +1,92 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mock.smtp.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class OperatorTest {
+
+    @Nested
+    class ContainsTest {
+        @Test
+        void containsShouldReturnTrueWhenTestedValuesContainsReferenceValue() {
```

Comment:
I will play the devil advocate but why did you choose to test the `String::contains` method ?

Reply:
We were not testing the string contains method but the operator that we just wrote.

We believe specifying the behavior of "what our code does" is important, and the fact that it relies on String:: contains is just an implementation details.

Reply:
Also, you review a "no review" commit...

Reply:
Yes sorry you commented on something that was from an other PR and that has been changed and merged yesterday. I just rebased and you will see that this commit disappeared